dsa-solutions/gfg-solutions/0004-fibbonacci-sum.md

```diff
@@ -0,0 +1,269 @@
+---
+id: fibonacci-sum
+title: Fibonacci Sum (Geeks for Geeks)
+sidebar_label: 0004 - Fibonacci Sum
+tags:
+  - intermediate
+  - Fibonacci
+  - Dynamic Programming
+  - Mathematics
+  - Algorithms
+---
+
+This tutorial contains a complete walk-through of the Fibonacci Sum problem from the Geeks for Geeks website. It features the implementation of the solution code in three programming languages: Python, C++, and Java.
+
+## Problem Description
+
+Given a positive number N, find the value of f0 + f1 + f2 + ... + fN where fi indicates the ith Fibonacci number. Note that f0 = 0, f1 = 1, f2 = 1, f3 = 2, f4 = 3, f5 = 5, and so on. Since the answer can be very large, return the result modulo 1000000007.
```

Comment:
Replace `f0 + f1 + f2 + ... + fN` to `$f0 + f1 + f2 + ... + fN$`

`f0 = 0, f1 = 1, f2 = 1, f3 = 2, f4 = 3, f5 = 5,` to `$f0 = 0, f1 = 1, f2 = 1, f3 = 2, f4 = 3, f5 = 5,$`

and `1000000007` to `$1000000007$`